docs/guide/java_serialization_guide.md

```diff
@@ -259,12 +254,11 @@ Fury fury=getFury();
 
 ### Security & Class Registration
 
-`FuryBuilder#requireClassRegistration` can be used to disable class registration, this will allow to deserialize objects unknown types, more flexible but **may be insecure if the classes contains malicious code**.
+`FuryBuilder#requireClassRegistration` can be used to disable class registration, this will allow to deserialize objects unknown types, more flexible but ****if the class contains malicious code, a security breach can occur**.**.
```

Comment:
```suggestion
`FuryBuilder#requireClassRegistration` can be used to disable class registration, this will allow to deserialize objects unknown types, more flexible but ****if the class contains malicious code, a security vulnerability can occur**.**.l
```